login-fire-social-behavior.html

```diff
@@ -78,6 +80,13 @@
      * method is called.
      */
     signInOrOut: function() {
+      // Skip processing of this method when we already have started a sign or signout process
+      if (this._inProgress) {
```

Comment:
This check must be done at the lowest level possible to avoid redo it several times. The `signIn` and `signOut` methods have to check the progress flag, not the callers of those methods.

Reply:
Agree. Will change it and move it to the low level methods.